### ISSUE TITLE
[Migrations] addColumn() does not work at all.

### ISSUE THREAD

**Issue by raphaelbgr** [agrosner/DBFlow#1654]
I'm unable to add a new column on an exsisting Database, the code executes fine, but when I pull my app .db file and open on the SQLite browser, it doesn't show my created columns.

I get this from the log:
```
class com.superrevendedores.data.db.AppDatabase$AddMagazineCoverToCycle executed successfully.
class com.superrevendedores.data.db.AppDatabase$AddCountryCodeForPhoneEntity executed successfully.
class com.superrevendedores.data.db.AppDatabase$AddSlugForBrandEntity executed successfully.
class com.superrevendedores.data.db.AppDatabase$AddOriginForSaleEntity executed successfully.
class com.superrevendedores.data.db.AppDatabase$AddOriginToClientEntity executed successfully.
```

DBFlow Version:
4.2.4

Bug or Feature Request:
For this to work, I have to call this code below:

```
FlowManager.reset();
FlowManager.destroy();
FlowManager.init(new FlowConfig.Builder(getApplicationContext()).build());
```

but by calling this I lose all my data, and I cannot afford to lose the internal db data.

Description:
How can I make Alter table migrations without losing my data?

Here is my migration code:

```
@Database(name = AppDatabase.NAME, version = AppDatabase.VERSION, foreignKeyConstraintsEnforced = true)
public class AppDatabase {

static final int VERSION = 55;

@Migration(version = 51, database = AppDatabase.class, priority = 1)
    public static class AddMagazineCoverToCycle extends AlterTableMigration<CycleEntity> {
        public AddMagazineCoverToCycle(Class<CycleEntity> table) {
            super(table);
        }

        @Override
        public void onPreMigrate() {
            addColumn(SQLiteType.TEXT, "magazineCover");
            super.onPreMigrate();
        }
    }

    @Migration(version = 52, database = AppDatabase.class, priority = 1)
    public static class AddCountryCodeForPhoneEntity extends AlterTableMigration<PhoneEntity> {
        public AddCountryCodeForPhoneEntity(Class<PhoneEntity> table) {
            super(table);
        }

        @Override
        public void onPreMigrate() {
            addColumn(SQLiteType.INTEGER, "countryCode");
            super.onPreMigrate();
        }
    }

    @Migration(version = 53, database = AppDatabase.class, priority = 1)
    public static class AddSlugForBrandEntity extends AlterTableMigration<BrandEntity> {
        public AddSlugForBrandEntity(Class<BrandEntity> table) {
            super(table);
        }

        @Override
        public void onPreMigrate() {
            addColumn(SQLiteType.TEXT, "slug");
            super.onPreMigrate();
        }
    }

    @Migration(version = 54, database = AppDatabase.class, priority = 1)
    public static class AddOriginForSaleEntity extends AlterTableMigration<SaleEntity> {
        public AddOriginForSaleEntity(Class<SaleEntity> table) {
            super(table);
        }

        @Override
        public void onPreMigrate() {
            addColumn(SQLiteType.TEXT, "origin");
            super.onPreMigrate();
        }
    }

    @Migration(version = 55, database = AppDatabase.class, priority = 1)
    public static class AddOriginToClientEntity extends BaseMigration {


        @Override
        public void migrate(@NonNull DatabaseWrapper database) {
            database.beginTransaction();
            String sql = "ALTER TABLE ClientEntity ADD COLUMN 'origin'";
            database.execSQL(sql);
            database.endTransaction();
            Timber.d("Migration [%s] SQL > %s", 55, sql);
        }
    }
}```

**Comment by raphaelbgr** [comment]
@agrosner , about migrations, do I need always to recreate de DB in order for them to be applied?

**Comment by raphaelbgr** [comment]
Forget all this. It's solved. The problem was that on Linux, the system doesn't say anything if the .db file has a lock (opened in anoter program), so always when I pulled the Database out of android I opened with SQLite Browser, but when saving another .db file replacing the one that was open on the SQLite Browser, even If I repoen the file, it wasn't getting the changes.

So always close a .db file before opening another.
